vizro-core/src/vizro/models/types.py

```diff
@@ -309,9 +312,26 @@ def wrapped(*args, **kwargs):
                     raise ValueError(f"{func.__name__} must supply a value to data_frame argument.") from exc
                 return captured_callable
 
+            return wrapped
+        elif self._mode == "aggrid":
```

Comment:
Is this identical to the `table` case? Can we put it in the same `elif self._mode in ["table", "ag_grid"]`?